src/hooks/Guilds/ether-swr/guild/useVotingResults.ts

```diff
@@ -1,4 +1,4 @@
-import { ERC20Info, useERC20Info } from '../erc20/useERC20Info';
+import { ERC20Info, useERC20Info } from '../../erc20/useERC20Info';

```

Comment:
Can you use absolute import?

Reply:
Yes! Have not gotten so far as to refactor the `guild`-hooks yet. Will also move the entire folder structure out of `ether-swr` so will fix it by then.